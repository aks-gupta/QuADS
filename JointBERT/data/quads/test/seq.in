is cultivator used for harvesting crops
is plough used during harvesting or seeding
for how many years is harrow usable
how is it known that a tractor is in good condition
how does seed drill work
what is repair cost for agricultural machines
what is the repair cost of plough
is produce sorter used for harvesting
what is the advantage of using cultivator
which type of cultivator is best
how much money is required for sprayer maintenance
can harrow be used for soil preparation
how can i tell if a tractor is in good condition
how to spray the emulsion
how to repair manure spreader
what is the use of roller
how does a planter works
how produce spreader is different from other tools
what is the life span of produce sorter
what is the advantage of using planter for sowing seeds
can chisel plow be used for all soil types
what is the best brand of manure spreader
is rotator cheap
is seed drill beneficial for sowing seeds
is manure spreader used in sowing
what is the life span of a cultivator
is a subsoiler safe
does using plough cause danger
what are the functions of rotator
is it important to purchase a sprayer
what is the maintenancecostof rotator
how does the chisel plow work
what does it mean to combine in farming
does using liquid manure require maintenance
can rotator be used safely
which brand is the best for roller
brand recommendation for chisel plow
in what stage of farming is roller used
what are the functions of seed drill
what are the types of chisel plow
how often should the sprayers be used for a crop
is using a combine dangerous
what are the functions of combine
what is the function of a disc harrow
what are the types of agricultural machines
is produce sorter essential for cultivation
what is the use of chisel plow in cultivation
how to use the harrow
what is the quantity needed for liquid manure
what are the examples of farm equipment
what is chisel plow
what is the maintenance cost of combine
working of a combine
what is a produce sorter
how expensive is harrow
how much maintenance required for planter
what is the cost of subsoiler
how much money is needed for maintaining a tractor
what are the priorities for mechanization
what are the types of manure spreader
how much can i buy peat soil for
what are best suited storage conditions for crops
which manure is best suited for black soil
can we store different crops together
is manure cheap
what are the various soil types
does my land need to near a well for good irrigation
what is the duration of ploughing
is soil preparation cheaper
in which season is harvesting done
is soil fertility improved by manuring
which month should i prepare soil for growing maize
why is sowing important
what is the price of manure
what are the best storage practices
which crop is suitable for transplant sowing of seeds
what is the importance of sowing
what are the advantages of manure
does manure have an expiration date
how cheap is peat
what are the kinds of sowing
what is red soil
what is the initial investment for sowing
how long does manuring phase last
what is crops can be grown on alluvial soil
how much rainfall is needed for irrigation
what are the advantages of surface irrigation systems
what is transplant sowing
what are the advantages of levelling for soil preparation
are there specific steps for soil preparation
what are the advantages of dibbing technique
what is the selling price of harvested crops
what are the different kinds of processes in agriculture
what are the best practices for sowing
how much investment is required to install a pusa bin
what is the process of ploughing
how long do sowed seeds take to grow
how long can we store grains
what type of soil is best for flowers
can i sow crops in different ways
where can i purchase red soil
which crops need additional attention during storage
is manuring beneficial
how to protect produce during storage
how to do effective harvesting
which irrigation technique conserves highest amount of water
how does silo compare to plow
how to improve storage conditions in india
what is lift irrigation system
is sowing seeds safe
is effective harvesting practice profitable
what are the disadvantages of soil preparation of red soil
what is the helpline for harvesting
how is black soil different from red soil
how much investment is required to install a silo storage
is there any specific temperature requirement for storage of grains
is dropping off the seed behind the plough effective
how to control storage temperature
what are the advantages of soil preparation of red soil
how much water is required for irrigating 2 hectares of land
name the components of fungicide
explain working of pesticides
lifetime of pesticide
which chemicals are there in fertilizers
examples of natural herbicide
what quantity of pesticides is required
how does fertilizer work
what are the most harmful chemicals in pesticides
can insecticides be bad for plants
what is natural pesticide
what is the most widely used insecticide
when should fertilizers be applied to plants
how much investment does fertilizer need
what is quality control for fertilizers
which pesticides is mostly used in india
what are zinc deficiency symptoms and what fertilizers contain zinc
can you use expired insecticide
disadvantage of using artificial fertilizer
name some types of herbicides
which variety of fertilizer is the best
what is fertilizer used for
is fertilizer expensive
which is most common element in pesticide
can we sell fungicides below market price
how much do pesticides cost
how much does pesticide cost per acre
how much do herbicides cost
which crops has highest consumption of pesticides
what is the cost of herbicide
where to buy cheap pesticide
best formula for fertilizer
how are herbicides used in agriculture
can pesticides cause harm
what are the example of organic insecticide
do pesticides have chemicals
explain herbicides
which fertilizer has potassium
best combination of pesticides
what is the shelf life of pesticides
where are fungicides used
how long does pesticide last
how to purchase cheap pesticides
how is fertilizer applied
what is a fertilizer
what is the value of fertilizer
are organic pesticides cheap
amount of fertilizer required
where to get cheap fungicides
is fertilizer good for the soil
are chemical fertilizers harmful for crops if used in excess quantity or in wrong combination of nutrients
how do insecticides protect crops
are pesticides used to control pests
what is herbicide used for
why is the price of roundup pesticide so high
can fungicides be bad
what elements should not be used while composting manure
where are fungicides used in crop production
what is the chemical composition of fungicides
what is the main ingredient in insecticide
what is the amount of herbicide used at one time
how and when are carrots harvested
what is the lifespan of organic carrot
where can i get information regarding horticulture
is viticulture a viable source of income
what should be the frequency of fertilizer application for potatoes
is the harvest for organic farming good
what is the best climate to grow apples
how to identify suitable land for grape farming
is there anyone i can contact to aid me in floriculture
are cherries and grapes different
is winegrowing an alternate name for viticulture
are fruits easy to grow
how often should a vegetable garden be watered
is viticulture an alternative farming method
what vegetables cannot be planted together
what are the disease and pests on carrot plants
what are the benefits of organic farming
does horticulture require special tools
can floriculture be harmful
how long does it take for potatoes to sprout
how should vegetable crops be fertilized
is viticulture cheap
what is the cost to purchase organic farming tools
which vegetables and fruits can be grown using organic farming
what are the types of flowers grown for floriculture
what vegetables to plant and when or what months can one plant vegetables
will i be able to grow flowers in my garden
how many types of horticulture are there
what is needed for organic farming
what are the soil conditions required for growing grapes
what are the requirements for successful floriculture business
what is the area in floriculture in india
is horticulture a cost effective farming method
how to do weed control for marigold plants
what is organic agriculture
what is the soil requirement for different types of floriculture
is there a website to buy organic farming tools
is it worth investing in floriculture
what is marigold floriculture
what are the soil conditions required for growing flowers
how to select location for viticulture
is the equipment required for floriculture expensive
what is horticultures importance
can horticulture be done in 1 hectare of land
what plants are grown in rotation to potato plants
what are the popular vegetables grown in horticulture
what are some of the organic pesticides used for farming
how do organic farms manage fertility
what vegetables are harder to grow
where to get information about floriculture
what type of soil is required for growing carrots
what is vegetable production or vegetable farming
whom to contact for horticulture starting
how many types of vegetables are there in the world
what is the soil fertility required for flower farming
what are the types of grapes in viticulture
what are the different type of nuts in india
how long does it take for vegetables to grow
does viticulture require special tools
can vegetables be raised for seed production
how much do silkworms cost
how long do pigs take to reproduce
what is the offspring of a hen called
is pig farming cheap or expensive
is goat milk expensive
is fish offsprings easy to make
what do silkworms need for nutrition
what do hens eat
to cows give milk throughout their life
why is pig farming bad
what is the difference between alligator farming and fish farming
how long do pigs live for
what is the price of maintaining a buffalo
which is the best quality silk
what is the price of silk sarees
is alligator farming easy to practice
do fish give any products
what do pigs give
what is the best way to farm fish
are birds easy to reproduce
how much does a pig weigh
what is the importance of pig farming
how do pigs reproduce
do goats get sick
how much does a turkey weigh
what is market price of bees
what is the selling price of a pig
is fish farming a seasonal type of farming
where is the best place to find information about pigs
what is the lifespan of a pig
how does a silkworm give silk
where can i buy alligators from
what is alligator farming
what is bee farming
is sericulture too expensive
what are the benefits of fish farming
which hen gives the best chicken meat
what do alligators eat
is alligator skin very costly
what is the difference between cheese and butter
are honey combs edible
is pig farming better than sheep farming
how much investment does a cow need
are alligators found in india
which sheep's wool is the best
is clarified butter ghee
any contacts for fish farming
what is the cost of keeping hens
is pig farming costly
how to maintain hygiene of a pig
why is poultry eaten
what is the best type of milk produce
what body parts of alligators are sold
how much investment do turkies require
is it costly to go for fish farming
what are the weather conditions required to keep birds
is pork from pig
how to make pigs breed
what is the diet of fish
how much milk does goat give
what is the average dry fruit farm yield
is there a website about vegetables
which spices grow the fastest
how much potato can be produced by a plant
why are vegetables so expensive
what pulse is liked by most people
is rain important to grow fruits
how much investment to start growing wheat
is growing dry fruits good for profit
what is the span of carrot farming
can babies consume grains
do millets need dry weather to grow
which is the best season to grow moong
how much does it cost to farm mangoes
how quickly can i sell spices
which soil is best to grow pulses
how much yield can i get from growing pulses
can pulses be grown in fields
what are the different varieties of mangoes
how much investment to be done to grow vegetables
are grains important to the environment
what pulses can be grown in 1 hectare of land
where can i get information about fruits
what is the cost price of growing dry fruits
which state consumes most wheat
is growing onions good for profit
which dry fruits can be grown in september
what is the shelflife of pulses
are dry fruits healthy
can mangoes grow in humid weather
in which soil is toor dal usually grown
can oil seeds be grown without special tools
do people like fruits
does january have a special pulse
what are the online portals to sell grains
what is the tastiest oil seed
how many months do bananas take to grow
in what temperature does toor grow best
how much yield from one acre of rice land
can wheat be grown only in summer
which is the best temperature for walnuts
which soil is best to grow oil seeds
can pulses be grown in the garden
when is the best time to harvest vegetables
when is the best time to harvest grains
why are my orange farms not yielding enough
are potatoes good for farming
is red soil suitable for walnut cultivation
where can i get information about grains
which fruits can be grown in september
is toor dal easy to farm
which grains can be grown in summer
how much yield do beans give for each acre of land
what are the health benefits of dry fruits
what diseases can grains get
how quickly can i sell vegetables
what all oil seeds are grown in india
is growing millets expensive
why are fruits so expensive
which vegetables can be grown in september