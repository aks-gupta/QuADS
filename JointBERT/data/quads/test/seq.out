O B-machinery_type B-machinery_agricultural_process O B-machinery_agricultural_process O
O B-machinery_type B-machinery_agricultural_process O B-machinery_agricultural_process O B-machinery_agricultural_process
O O O B-machinery_shelf_life O B-machinery_type O
O O O O O O B-machinery_type O O B-machinery_general_info B-machinery_shelf_life
O O B-machinery_agricultural_process O B-machinery_general_info
O O B-machinery_maintenance_cost B-machinery_price O O B-machinery_type
O O O B-machinery_maintenance_cost B-machinery_price O B-machinery_type
O O O B-machinery_agricultural_process O B-machinery_agricultural_process
O O O B-machinery_general_info O O B-machinery_type
O B-machinery_subtype O B-machinery_type O B-machinery_general_info
O O B-machinery_price O O O B-machinery_type B-machinery_maintenance_cost
O B-machinery_type O B-machinery_agricultural_process O B-machinery_agricultural_process O
O O O O O O B-machinery_type O O B-machinery_general_info B-machinery_shelf_life
O O O O O
O O B-machinery_maintenance_cost O O
O O O B-machinery_agricultural_process O B-machinery_type
O O O B-machinery_type B-machinery_general_info
O O O O B-machinery_subtype O O B-machinery_type
O O O B-machinery_shelf_life B-machinery_shelf_life O O O
O O O B-machinery_general_info O O B-machinery_type O B-machinery_agricultural_process B-machinery_agricultural_process
O O O O B-machinery_agricultural_process O O B-machinery_agricultural_process B-machinery_subtype
O O O B-machinery_general_info B-machinery_general_info O O O
O B-machinery_type B-machinery_price
O B-machinery_agricultural_process O O O B-machinery_agricultural_process B-machinery_agricultural_process
O O O B-machinery_agricultural_process O B-machinery_agricultural_process
O O O B-machinery_shelf_life B-machinery_shelf_life O O B-machinery_type
O O B-machinery_type B-machinery_general_info
O O B-machinery_type B-machinery_agricultural_process B-machinery_general_info
O O O B-machinery_agricultural_process O B-machinery_type
O O B-machinery_general_info O B-machinery_price O B-machinery_type
O O O B-machinery_price B-machinery_type
O O O O O B-machinery_general_info
O O O O O O O O
O O O O O B-machinery_maintenance_cost
O B-machinery_type O B-machinery_agricultural_process B-machinery_general_info
O B-machinery_general_info O O B-machinery_general_info O B-machinery_type
B-machinery_general_info O O O O
O O O O O O B-machinery_type B-machinery_agricultural_process
O O O B-machinery_agricultural_process O B-machinery_agricultural_process O
O O O B-machinery_subtype O O O
O B-machinery_shelf_life O O B-machinery_type O B-machinery_agricultural_process O O O
O O O O B-machinery_general_info
O O O B-machinery_agricultural_process O O
O O O B-machinery_agricultural_process O O B-machinery_subtype B-machinery_type
O O O B-machinery_subtype O O B-machinery_type
O O O O O B-machinery_agricultural_process
O O O B-machinery_agricultural_process O O O O B-machinery_agricultural_process
O O B-machinery_agricultural_process O B-machinery_type
O O O B-machinery_general_info O O O O
O O O B-machinery_subtype O O B-machinery_type
O O O O
O O O B-machinery_maintenance_cost B-machinery_price O O
B-machinery_general_info O O O
O O O O O
O B-machinery_price O B-machinery_type
O O B-machinery_maintenance_cost O O B-machinery_type
O O O B-machinery_price O B-machinery_type
O O B-machinery_price O O O B-machinery_maintenance_cost O B-machinery_type
O O O O O B-machinery_type
O O O B-machinery_subtype O O O
O O O O B-land_cost B-land_soil_type B-land_soil_type O
O O B-agricultural_process_adv O B-agricultural_process_type O O O
O B-agricultural_process_type O B-agricultural_process_adv O O B-land_soil_type B-land_soil_type
O O B-agricultural_process_type O O O
O B-agricultural_process_type B-land_cost
O O O O B-land_soil_type B-agricultural_process_type
O O B-land_area B-agricultural_process_adv O O O O O B-agricultural_process_adv B-agricultural_process_type
O O O B-agriculture_process_duration O B-agricultural_process_sub_type
O B-land_soil_type B-agricultural_process_type B-land_cost
O O B-agriculture_process_duration O B-agricultural_process_type O
O B-land_soil_type B-agricultural_process_requirement B-agricultural_process_adv O B-agricultural_process_type
O B-agriculture_process_duration O O O B-land_soil_type O B-agriculture_process_duration O
O O B-agricultural_process_type B-agricultural_process_adv
O O O B-land_cost O B-agricultural_process_type
O O O B-agricultural_process_adv B-agricultural_process_type O
O O O O O B-agricultural_process_sub_type B-agricultural_process_type O B-agricultural_process_type
O O O O O B-agricultural_process_type
O O O B-agricultural_process_adv O B-agricultural_process_type
O B-agricultural_process_type O O B-agriculture_process_duration O
O B-land_cost O B-land_soil_type
O O O B-agricultural_process_type O B-agricultural_process_type
O O B-land_soil_type B-land_soil_type
O O O O B-land_cost O B-agricultural_process_type
O B-agriculture_process_duration O B-agricultural_process_type O O
O O O O O B-agriculture_process_duration O B-land_soil_type B-land_soil_type
O O O O B-agricultural_process_adv O B-agricultural_process_type
O O O B-agricultural_process_adv O O B-agricultural_process_type O
O O B-agricultural_process_sub_type B-agricultural_process_type
O O O B-agricultural_process_adv O B-agricultural_process_sub_type O B-land_soil_type B-agricultural_process_type
O O O B-agricultural_process_requirement O B-land_soil_type B-agricultural_process_type
O O O B-agricultural_process_adv O O B-agricultural_process_requirement
O O O O B-land_cost O B-agricultural_process_type O
O O O O B-agricultural_process_type O B-agricultural_process_type O O
O O O B-agricultural_process_adv O O B-agricultural_process_type
O O B-land_cost O B-land_soil_type O O O O O
O O O B-agricultural_process_type O B-agricultural_process_sub_type
O B-agriculture_process_duration O B-agricultural_process_type B-agricultural_process_type O O B-agriculture_process_duration
O B-agriculture_process_duration O O B-agricultural_process_type O
O B-agricultural_process_type O B-land_soil_type O B-agricultural_process_adv O O
O O B-agricultural_process_type O O O O
O O O B-land_cost B-land_soil_type B-land_soil_type
O O B-agricultural_process_adv O B-agricultural_process_requirement O B-agricultural_process_type
O B-agricultural_process_type B-agricultural_process_adv
O O B-agricultural_process_requirement B-agricultural_process_output O B-agricultural_process_type
O O O B-agricultural_process_disadv B-agricultural_process_type
O B-agricultural_process_type B-agricultural_process_requirement B-agricultural_process_adv O B-land_cost O B-agricultural_process_requirement
O O B-agricultural_process_sub_type O O B-agricultural_process_sub_type
O O B-agricultural_process_adv B-agricultural_process_type O O O
O O B-agricultural_process_sub_type B-agricultural_process_type O
O B-agricultural_process_type B-agricultural_process_type B-agricultural_process_adv
O B-agricultural_process_disadv B-agricultural_process_type O B-land_cost
O O O B-agricultural_process_adv O B-land_soil_type B-agricultural_process_type O B-land_soil_type B-land_soil_type
O O O B-agricultural_process_adv O B-agricultural_process_type
O O B-land_soil_type B-land_soil_type O O B-land_soil_type B-land_soil_type
O O B-land_cost O B-land_soil_type O O O B-agricultural_process_sub_type B-agricultural_process_type
O O O O B-agricultural_process_requirement B-agricultural_process_requirement O B-agricultural_process_type O O
O O O O B-agricultural_process_type O O B-agricultural_process_sub_type B-agricultural_process_disadv
O O O B-agricultural_process_type B-agricultural_process_requirement
O O O B-agricultural_process_adv O B-land_soil_type B-agricultural_process_type O B-land_soil_type B-land_soil_type
O O B-agricultural_process_requirement O B-land_soil_type O O O B-land_area O B-land_area
O O B-agrochemical_ingredients O B-agrochemical_type
O B-agrochemical_usecase O B-agrochemical_type
B-agrochemical_shelflife O B-agrochemical_type
O B-agrochemical_ingredients O O O B-agrochemical_type
O O B-agrochemical_usecase B-agrochemical_type
O B-agrochemical_dosage O B-agrochemical_type O O
O O B-agrochemical_type B-agrochemical_usecase
O O O O B-agrochemical_disadv B-agrochemical_ingredients O B-agrochemical_type
O B-agrochemical_type O B-agrochemical_disadv O O
O O B-agrochemical_usecase B-agrochemical_type
O O O O O B-agrochemical_usecase B-agrochemical_type
O O B-agrochemical_type O O O O
O B-agrochemical_dosage B-agrochemical_price O B-agrochemical_type O
O O O B-agrochemical_usecase O B-agrochemical_type
O B-agrochemical_type O O B-agrochemical_usecase O O
O O B-agrochemical_ingredients O O O O B-agrochemical_type B-agrochemical_ingredients B-agrochemical_ingredients
O O B-agrochemical_usecase O B-agrochemical_type
B-agrochemical_adv O O B-agrochemical_usecase B-agrochemical_type
O O O O B-agrochemical_type
O B-agrochemical_ingredients O B-agrochemical_type O O O
O O B-agrochemical_type B-agrochemical_usecase O
O B-agrochemical_type B-agrochemical_price
O O O O B-agrochemical_ingredients O B-agrochemical_type
O O O B-agrochemical_type O O B-agrochemical_price
O B-agrochemical_dosage O B-agrochemical_type B-agrochemical_price
O B-agrochemical_dosage O B-agrochemical_type B-agrochemical_price O O
O B-agrochemical_dosage O B-agrochemical_type B-agrochemical_price
O B-agrochemical_usecase O O O O B-agrochemical_type
O O O B-agrochemical_price O B-agrochemical_type
O O B-agrochemical_price B-agrochemical_price B-agrochemical_type
O B-agrochemical_ingredients O B-agrochemical_type
O O B-agrochemical_type B-agrochemical_usecase O O
O B-agrochemical_type B-agrochemical_usecase B-agrochemical_disadv
O O O O O B-agrochemical_usecase B-agrochemical_type
O B-agrochemical_type O B-agrochemical_ingredients
O B-agrochemical_type
O B-agrochemical_type O B-agrochemical_ingredients
O B-agrochemical_ingredients O B-agrochemical_type
O O O O B-agrochemical_shelflife O B-agrochemical_type
O O B-agrochemical_type B-agrochemical_usecase
O O O B-agrochemical_type B-agrochemical_shelflife
O O B-agrochemical_price B-agrochemical_price B-agrochemical_type
O O B-agrochemical_type O
O O O B-agrochemical_type
O O O B-agrochemical_price O B-agrochemical_type
O B-agrochemical_usecase B-agrochemical_type B-agrochemical_price
B-agrochemical_price O B-agrochemical_type O
O O O B-agrochemical_price B-agrochemical_type
O B-agrochemical_type B-agrochemical_adv O O O
O B-agrochemical_ingredients B-agrochemical_type B-agrochemical_disadv O B-agrochemical_usecase O B-agrochemical_usecase O O B-agrochemical_dosage O O O B-agrochemical_ingredients O O
O O B-agrochemical_type B-agrochemical_usecase B-agrochemical_usecase
O B-agrochemical_type B-agrochemical_usecase O B-agrochemical_usecase B-agrochemical_type
O O B-agrochemical_type B-agrochemical_usecase O
O O O B-agrochemical_price O O B-agrochemical_type O O
O B-agrochemical_type O B-agrochemical_disadv
O B-agrochemical_ingredients O O O B-agrochemical_usecase O B-agrochemical_ingredients O
O O B-agrochemical_type B-agrochemical_usecase O B-agrochemical_usecase O
O O O B-agrochemical_ingredients B-agrochemical_ingredients O B-agrochemical_type
O O O O B-agrochemical_ingredients O B-agrochemical_type
O O O B-agrochemical_price O B-agrochemical_type B-agrochemical_usecase O O B-agrochemical_shelflife
O O O O B-method_subtype B-method_yield
O O O B-method_duration O B-method_type B-method_subtype
O O O O B-method_helpline O B-method_type
O B-method_type O O O O O
O O O O O O B-method_requirements O O B-method_subtype
O O B-method_yield O B-method_type O B-method_adv
O O O B-method_adv B-method_requirements O B-method_duration O
O O O O B-method_requirements O B-method_type O
O O O O O B-method_helpline O O O O B-method_type
O O O B-method_type O
O B-method_duration O O O O B-method_type
O B-method_subtype O O B-method_duration
O O O O B-method_subtype O O B-method_requirements
O B-method_type O O O O
O B-method_subtype O O O O
O O O B-method_requirements O B-method_requirements O B-method_subtype O
O O O B-method_adv O B-method_type O
O B-method_type B-method_requirements O O
O B-method_type O B-method_disadv
O B-method_duration O O O O B-method_subtype O O
O O B-method_subtype O O B-method_requirements
O B-method_type O
O O O B-method_investment_cost O B-method_investment_cost B-method_type O O
O B-method_subtype O B-method_subtype O O B-method_duration O B-method_type O
O O O O O B-method_type B-method_duration O B-method_type
O B-method_subtype O O O O O O B-method_duration O O O B-method_subtype
O O O O O B-method_duration B-method_type O O O
O O O O B-method_type O O
O O B-method_adv O B-method_type O
O O O B-method_requirements O B-method_requirements O B-method_duration B-method_type
O O O B-method_requirements O O B-method_type O
O O O B-method_requirements O B-method_type O O
O B-method_type O B-method_investment_cost B-method_disadv O O
O O O B-method_requirements O O B-method_subtype O
O O B-method_type B-method_type
O O O B-method_requirements B-method_requirements O O O O B-method_type
O O O B-method_helpline O B-method_investment_cost B-method_type O O
O O O O O B-method_type
O O B-method_subtype B-method_type
O O O B-method_requirements O B-method_requirements O B-method_duration B-method_type
O O O O O B-method_type
O O O B-method_requirements O B-method_type O
O O B-method_type O
O B-method_type O O O O O O B-method_requirements
O O O B-method_duration O O O B-method_subtype O
O O O O B-method_subtype B-method_duration O B-method_type
O O O O O B-method_type B-method_requirements B-method_use O O
O O B-method_type O B-method_use B-method_requirements
O B-method_subtype O O O B-method_duration
O O O B-method_helpline O B-method_type
O O O B-method_requirements O B-method_requirements O B-method_duration B-method_subtype
O O B-method_subtype B-method_use O B-method_subtype O
O O B-method_helpline O B-method_type O
O O O O B-method_subtype O O O O O
O O O B-method_requirements B-method_requirements B-method_requirements O B-method_type O
O O O O O B-method_type O B-method_type
O O O O O O O O O
O B-method_duration O O O O B-method_subtype O B-method_duration
O B-method_type B-method_requirements O O
O B-method_subtype O O O B-method_requirements B-method_use
O O O B-animal_type B-animal_cost_price
O O O B-animal_type O O B-animal_reproduction
O O O B-animal_reproduction O O B-animal_type O
O B-animal_type O B-animal_cost_price O B-animal_cost_price
O B-animal_type B-animal_byproduct B-animal_cost_price
O B-animal_type B-animal_reproduction O O O
O O B-animal_type O O B-animal_diet
O O B-animal_type B-animal_diet
O B-animal_type B-animal_byproduct B-animal_byproduct O O B-animal_lifespan
O O B-animal_type O O
O O O O O B-animal_type O O B-animal_type O
O O O B-animal_type O O
O O O B-animal_selling_price O O O B-animal_type
O O O O O B-animal_byproduct
O O O B-animal_selling_price O B-animal_byproduct O
O B-animal_type O O O O
O B-animal_type B-animal_byproduct O B-animal_byproduct
O O B-animal_type B-animal_byproduct
O O O O O O O B-animal_type
O O O O B-animal_reproduction
O O O O B-animal_type B-animal_weight
O O O O O B-animal_type O
O O B-animal_type B-animal_reproduction
O B-animal_type O B-animal_diseases
O O O O B-animal_type B-animal_weight
O O B-animal_selling_price B-animal_selling_price O B-animal_type
O O O B-animal_selling_price B-animal_selling_price O O B-animal_type
O B-animal_type O O O O O O
O O O O O O O O O B-animal_type
O O O B-animal_lifespan O O B-animal_type
O O O B-animal_type B-animal_byproduct B-animal_byproduct
O O O O B-animal_type O
O O B-animal_type O
O O B-animal_type O
O O O B-animal_cost_price
O O O O O B-animal_type O
O B-animal_type B-animal_byproduct O O B-animal_type B-animal_diet
O O B-animal_type B-animal_diet
O B-animal_type B-animal_byproduct O B-animal_cost_price
O O O O O B-animal_byproduct O O
O B-animal_byproduct O O
O B-animal_type O O O B-animal_type O
O O O O O B-animal_type O
O B-animal_type O O O
O B-animal_type B-animal_byproduct O O O
O O O O
O O O B-animal_type O
O O O B-animal_cost_price O O B-animal_type
O B-animal_type O B-animal_cost_price
O O O B-animal_maintenance O O B-animal_type
O O O B-animal_diet
O O O O O O B-animal_byproduct O
O O O O B-animal_type O O
O O O O B-animal_type O
O O B-animal_cost_price O O O B-animal_type O
O O O B-animal_diet B-animal_diseases O O O O
O B-animal_byproduct O B-animal_type
O O O B-animal_type B-animal_reproduction
O O O B-animal_diet O B-animal_type
O O B-animal_byproduct O B-animal_type B-animal_byproduct
O O O O O B-crop_name O O
O O O O O B-crop_name
O O B-crop_growth_time O O
O O B-crop_name O O O O O O
O O B-crop_name O B-crop_cost_price
O O O O O O O
O B-crop_rainfall O O B-crop_growth_time B-crop_name
O O B-crop_cost_price O O B-crop_growth_time B-crop_name
O B-crop_growth_time O B-crop_name O O O
O O O B-crop_growth_time O B-crop_name O
O O B-crop_selling_price B-crop_rainfall
O B-crop_name O O B-crop_weather O B-crop_growth_time
O O O O O O B-crop_growth_time O
O O O O B-crop_cost_price O O B-crop_name
O O O O B-crop_selling_price O
O B-crop_soil_type O O O B-crop_growth_time O
O O O O O O O B-crop_growth_time O
O O O B-crop_growth_time O O
O O O O O O B-crop_name
O O B-crop_cost_price O O O O B-crop_growth_time B-crop_name
O B-crop_rainfall O O O O
O O O O B-crop_growth_time O O O O O
O O O O O O B-crop_name
O O O B-crop_cost_price B-crop_name O B-crop_growth_time O B-crop_name
O O B-crop_selling_price O B-crop_name
O B-crop_growth_time B-crop_name O O O
O O B-crop_name O O B-crop_growth_time O O
O O O B-crop_growth_time O O
O O B-crop_name O
O B-crop_name B-crop_growth_time O B-crop_humidity B-crop_weather
O O B-crop_soil_type O O O O B-crop_growth_time
O O O O B-crop_growth_time O O O
O O O B-crop_name
O O O O O O
O O O O O O B-crop_selling_price B-crop_rainfall
O O O O O O
O O O O B-crop_name O O B-crop_growth_time
O O B-crop_temperature O O B-crop_growth_time O
O O O O O O O B-crop_name O
O B-crop_name O B-crop_growth_time O O O
O O O O B-crop_temperature O B-crop_name
O B-crop_soil_type O O O B-crop_growth_time O O
O O O B-crop_growth_time O O O
O O O O B-crop_growth_time O O B-crop_name
O O O O B-crop_growth_time O O B-crop_rainfall
O O O O O O O O
O B-crop_name O O O
O B-crop_soil_type B-crop_soil_type O O B-crop_name O
O O O O O O B-crop_rainfall
O B-crop_name O O B-crop_growth_time O O
O O O O O O
O B-crop_rainfall O O B-crop_growth_time O O
O O O O B-crop_name O O O O O O
O O O O O O O B-crop_name
O O O B-crop_rainfall O
O O O O B-crop_selling_price B-crop_name
O O O O O B-crop_growth_time O O
O B-crop_growth_time B-crop_name B-crop_cost_price
O O B-crop_name O B-crop_cost_price
O B-crop_name O O B-crop_growth_time O O